### ISSUE TITLE
MS Teams does not populate channel with history

### ISSUE THREAD

**Issue by kosi2801** [ramboxapp/community-edition#1641]
The MS Teams view does not load the history of a channel if there is one.

### Steps to reproduce

1. Log in to MS Teams
2. Open any channel in any team
3. see the "Welcome to the team!" message/screen, no other history or older conversation

### Expected behavior

If there was already activity in the channel this history should be loaded and displayed correctly,

### Actual behavior

Everytime a channel is selected for the first time in a Rambox session it appears as if it had never seen any activity. All activity which happens AFTER the Rambox session has been started is displayed and added correctly from that point on, as if it were the first activity in that channel.

See screenshot comparing the display in the web-view of Teams in Firefox (left, 52.7.3 ESR (64-bit)) vs. in Rambox (right, 0.5.17): ![history_issue](https://user-images.githubusercontent.com/1632470/39752682-786ec9f6-52bc-11e8-96e6-4e37db3af822.png)

### ENVIRONMENT
Version: 0.5.17
Platform: linux (x64)
Electron: 1.8.4
Chromium: 59.0.3071.115
Node: 8.2.1

**Comment by Ash258** [comment]
Try to log into MS teams from other browser / private window if there is same behaviour.

**Comment by kosi2801** [comment]
Linux FF 52.7.3 ESR (64-bit) Private Mode: correct behavior, history shows up
Linux Chromium: "Hmm. Your browser version isn't supported."
Win7 Chrome 66.0.3359.139 (Official Build) (64-bit): correct behavior, history shows up
Win7 Chrome 66.0.3359.139 (Official Build) (64-bit), Private Mode: correct behavior, history shows up
Win7 IE 11 (64-bit): correct behavior, history shows up
Win7 FF 59.0.3 (64-Bit): correct behavior, history shows up
Win7 FF 59.0.3 (64-Bit) Private Mode: correct behavior, history shows up

Seems fine in all browsers I have at hand.

I have NO custom code set up for this service in Rambox.
In the developer tools in Rambox opened the Console while reloading the Teams service.
There was one error (besides the initial "USER NOT  LOGGED IN" and some "Start loading..." messages):
`index.html:63 GET file:///opt/Rambox/resources/app.asar/ext/packages/ext-locale/build/ext-locale-en.js net::ERR_FILE_NOT_FOUND`

Maybe this is a useful hint? If not, how can I give you more information on the issue?

**Comment by kosi2801** [comment]
New info, I maybe worked around this issue temporarily.

I noticed that for other participants in my Teams channels the UI, notifications and some common channel names were displayed in German. On my UI through Rambox everything was displayed in English.

I changed the language setting in Rambox but that didn't change anything. I then also read in another issue posted for Teams here that things could change when Teams was removed from Rambox and re-added.

I did that by 

1. removing Teams from the UI 
2. stopping Rambox 
3. deleting the ~/.config/Rambox/Partitions/teams_1 directory
4. restarting Rambox
5. re-adding and re-login Teams

Suddenly all messages and notifications from Teams are displayed in German and the history seems to be complete so far.

What would have helped enormously during this would be an opion/action from within Rambox to dump all caches (&settings?) for a service and make a fresh start without having to perform aboves steps manually.
Also I noticed that the language setting can only be set globally, it would be helpful if that could be set by service.

I will keep an eye on this and wait a bit if the issue has been resolved permanently or reappears...
